functions/src/fetch.ts

```diff
@@ -1,9 +0,0 @@
-let fetchRequest: typeof fetch
```

Comment:
This file seems useful to have? Is there a better alternative to node-fetch

Reply:
No, `node-fetch` is fine if we actually need it, but right now we don't, so we definitely don't need "a funny shim for using either the built in thing we don't need, or the replacement package that we don't need."